docs/guide/web-simulation.md

```diff
@@ -54,13 +54,18 @@ Then, start a session and simulation servers:
 cd $WEBOTS_HOME/resources/web/server
 ./server.sh start default
 ```
+
+If you have installed Webots with the `.deb` or the `.snap` package, you should:
+ - Copy the server folder in a place where they can be launched without `sudo` and run the scripts from there.
```

Comment:
```suggestion
 - Copy the [server]({{ url.github_tree }}/resources/web/server) folder in a place where you have read/write rights, so that the server can be launched without administrator privileges.
```